genre/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name, id = Random.rand(1..1000))
+    @name = name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    item.genre = self
```

Comment:
* Kindly fix this error I get when I try to create my genre below ⬇️  is a screenshot of the error I get 
![image](https://user-images.githubusercontent.com/52497246/196911954-fec7a74a-640a-4695-8781-0c7fab117094.png)